### ISSUE TITLE
Feature Request: Import Theme Configuration?

### ISSUE THREAD

**Issue by ghost** [themix-project/oomox#23]
I'd like to request if possible an import theme button that can import the existing GTK3 based configurations in favor of an old theme.

Or something similar

I don't know if it's possible but please consider this as a suggestion.

Thanks

**Comment by actionless** [comment]
I did not understood clearly what does that mean, to import the colorscheme from another GTK theme?

**Comment by mikhail21393** [comment]
Yes exactly! I noticed that the configurations are not the same on GTK Theme or modified on some sort.

**Comment by actionless** [comment]
now there is a import plugin api feel free to implement it as plugin

ex.: https://github.com/themix-project/oomox/tree/master/plugins

**Comment by tnilzon** [comment]
Help needed:
How to implement this plugin in Oomox?
(I am on Majaro gnome, and installed Oomox via Pamac.)

**Comment by actionless** [comment]
@tnilzon you can take a look on this really tiny plugin which takes colors from `.yml` palettes from `base16` project and imports them as oomox themes:

https://github.com/themix-project/oomox/tree/master/plugins/import_base16

**Comment by actionless** [comment]
however, i think, it will be nice to add "import" button in GUI (because now for importing something you need to copy it to `~/.config/oomox/colors`)

**Comment by actionless** [comment]
but, just to clarify, i am going to implement only "Import" button itself, which will be used by import plugins

so if you want to import the colors from a new source you still need to implement that plugin itself (so far supported only importing colors from `base16` and from images (also there is import of `Xresources` palette done implicitly when using `xresources` color themes but it also going to be refactored to be a plugin)

**Comment by tnilzon** [comment]
Would this be possible: Press the 'Import' button -> Select a gtk theme -> the selected theme is then imported and can be edited as a new theme (as the "default" pre-installed Oomox themes) ?

**Comment by tnilzon** [comment]
Well I am afraid I am not skilled to write any plugin by myself. So I will have to wait for it to be implemented (the button and the plugin).
Thanks anyway!

**Comment by actionless** [comment]
rather than waiting you could just use color picker in oomox when selecting the colors (it's available when you opening palette editor by clicking on button near each color) and pick the colors from the theme of your choice

**Comment by tnilzon** [comment]
OK, Thanks the tip!
